parm/config/config.anal

```diff
@@ -159,12 +159,12 @@ if [[ $RUN_ENVIR == "emc" ]]; then
 
 #   Turn off assimilation of S-NPP CrIS
     if [[ "$CDATE" -ge "2021052118" && "$CDATE" -lt "2021092206" ]]; then
-	export SATINFO=$FIXgsi/gfsv16_historical/global_satinfo.txt.2021052118
+	export SATINFO=$FIXgsi/gfsv16_historical/gfsv16.3/global_satinfo.txt.2021052118
```

Comment:
As above, Dogwood directory `/lfs/h2/emc/da/noscrub/emily.liu/git/global-workflow/gfs.v16.3.0/sorc/gsi.fd/fix/gfsv16_historical/gfsv16.3` does not contain `global_satinfo.txt.2021052118`.   It contains `global_satinfo.txt.2021092206` and `global_satinfo.txt.2021102612`.